src/data/polynomial/algebra_map.lean

```diff
@@ -204,6 +204,18 @@ lemma aeval_comp {A : Type*} [comm_semiring A] [algebra R A] (x : A) :
   aeval x (p.comp q) = (aeval (aeval x q) p) :=
 eval₂_comp (algebra_map R A)
 
+lemma aeval_list_sum (l : list R[X]) (x : A) :
```

Comment:
Do we need this lemma? Doesn't `map_list_sum (aeval x)` produce the same statement?

Reply:
We can use `map_list_sum (aeval x) l` here. But in `aeval_list_prod` we need to use `_root_.map_list_prod (aeval x) l` because there is a lemma `polynomial.map_list_prod`. If `map_list_sum (aeval x) l` is prefered, I will change them to this form.

Reply:
`polynomial.map_list_prod` should probably be `protected` to avoid that name clash, but that's out of scope for this PR